packages/font-downloader/font-downloader.pacscript

```diff
@@ -0,0 +1,19 @@
+name="font-downloader"
+gives="font-downloader"
+version="10.0.0"
+build_depends="cmake meson libhandy-1-dev gettext"
+# HACK: https://github.com/pacstall/pacstall/issues/727
+depends="libgtk-3-dev python3-gi"
+url="https://github.com/GustavoPeredo/Font-Downloader/archive/refs/tags/v${version}.zip"
+repology=("project: fontdownloader")
+description="Install fonts from online sources"
+hash="eeafd4ac9cb0d47fd0c1512e07805d0f7a639cdbbc688647249eaee8d1753e23"
+maintainer="সৌম্যদীপ <soumyadeepghosh2004@zohomail.in>"
```

Comment:
By the way @soumyatheman, you have read the notice about automatic issue assignments right?

Reply:
Yes I read it. The package maintainers will get auto assigned.

Reply:
No, you won't be auto-assigned if you don't use your GitHub username in the `maintainer` field. So, if you wish to be auto-assigned in the future, change your name from `সৌম্যদীপ` to `soumyatheman` in all the pacscripts.